metrics/machine_learning/tensorflow.sh

```diff
@@ -10,32 +10,65 @@ set -e
 SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
 source "${SCRIPT_PATH}/../lib/common.bash"
 
-TEST_NAME="tensorflow"
 IMAGE="docker.io/library/tensorflow:latest"
 DOCKERFILE="${SCRIPT_PATH}/tensorflow_dockerfile/Dockerfile"
-CMD="cd benchmarks/scripts/tf_cnn_benchmarks/ && python tf_cnn_benchmarks.py -data_format=NHWC --device cpu --batch_size 512 --num_batches=300"
+CMD_RUN="cd benchmarks/scripts/tf_cnn_benchmarks/ && python tf_cnn_benchmarks.py -data_format=NHWC --device cpu --batch_size 512 --num_batches=300 > result"
```

Comment:
For `--batch_size` and `--num_batches` could you replace the constants for global variables with default initial values?

Reply:
@dborquez done

Reply:
@dborquez changes applied